text/profiles/0239-profiles-data-model.md

```diff
@@ -1349,7 +1349,7 @@ sample:
     locations_length: 3
     value:
       - 100
-  - locations_start_index: 2
+  - locations_start_index: 3
```

Comment:
also, should below

```
location_indices:
  - 0 # foo
  - 1 # bar
  - 2 # baz
  - 4 # abc
  - 5 # def
```

be

```
location_indices:
  - 0 # foo
  - 1 # bar
  - 2 # baz
  - 3 # abc
  - 4 # def
```

Reply:
Good spot! :+1: I have fixed also the indices of `location_indices`.